index.html

```diff
@@ -8,41 +9,343 @@
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+        <div class="card-holder-item">
+
+            <div class="holder-image">
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+        <div class="card-holder-item">
+
+            <div class="holder-image">
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+        <div class="card-holder-item">
+
+            <div class="holder-image">
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" id="fourth-item">
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+
+    </div>
+
+    <div class="about-me">
+
+        <div id="title">
+            <h2>About me</h2>
+        </div>
+        <div class="about-me-body">
```

Comment:
Also, this should be a `<p>` tag, not a `<div>` tag that would be better if you do that